src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/payments/PaymentsPublisher.java

```diff
@@ -45,6 +45,12 @@ public void send(PaymentCommand cmd) {
             );
             message.setLabel(cmd.getLabel());
 
+            LOG.info("About to send message to payments queue. ID: {}, Label: {}, Content: {}",
+                    message.getMessageId(),
+                    message.getLabel(),
+                    messageContent
```

Comment:
Is there no secret information in the content? Just checking.

Reply:
it is already logging in after the call.

Reply:
alright, then